templates/rhpam74-authoring.yaml

```diff
@@ -883,13 +883,39 @@ objects:
           - name: RHPAM_PASSWORD
             value: "${KIE_SERVER_H2_PWD}"
           - name: RHPAM_XA_CONNECTION_PROPERTY_URL
-            value: "jdbc:h2:/opt/eap/standalone/data/rhpam"
+            value: "jdbc:h2:/deployments/data/h2/rhpam;AUTO_SERVER=TRUE"
           - name: RHPAM_SERVICE_HOST
             value: "dummy_ignored"
           - name: RHPAM_SERVICE_PORT
             value: "12345"
           - name: KIE_SERVER_PERSISTENCE_DIALECT
             value: "org.hibernate.dialect.H2Dialect"
+          - name: AUTO_CONFIGURE_EJB_TIMER
```

Comment:
Regarding this, make sure that there is a bug with the ejb timer integration, if so, remove this and fix it on jboss-kie-module instead.

Reply:
Sure, that's the reason I sent that email to you guys. The fix is documented in https://issues.jboss.org/browse/KIECLOUD-216 I'll force pushed once the fix got merged \o/